tutorials/fundamentals/shuffle.md

```diff
@@ -0,0 +1,28 @@
+To properly shuffle an array in JavaScript, use the Fisher-Yates shuffle algorithm.
+The algorithm randomly draws an element from the array until no elements remain in the array.
```

Comment:
"The algorithm loops through each element in the array and swaps it with a random element in the array as shown below."